app/views/shared/summary_cards/_resource.html.slim

```diff
@@ -1,66 +1,42 @@
 .summary-card
   .summary-card__body
-    .summary-card__row--half
-      h5.summary-card__title
-        span class="glyphicon glyphicon--right glyphicon-#{resource_icon(resource)}"
-        = link_to resource.title, resource
-        - if defined?(hit)
-          .summary-card__relevancy
-            = render 'shared/components/score', hit: hit 
-      .summary-card__top-actions
-        .summary-card__more.summary-card__margin-right
-          = render 'shared/components/btn_count', path: '#', icon: 'plus', label: '10', style: 'primary'
-        - if policy(resource).edit?
-          .summary-card__more.summary-card__margin-right
-            = link_to edit_resource_path(resource), class: 'btn btn-primary' do
-              span class="glyphicon  glyphicon-pencil"
-        - if policy(resource).destroy?
-          .summary-card__more
-            = link_to resource, method: 'DELETE', class: 'btn btn-default', data: { confirm: 'Are you sure?' } do
-              span class="glyphicon  glyphicon-remove"
+    .summary-card__row--double
+      div class="entity-icon entity-icon--#{resource.resource_type}"
+        i class="fa fa-#{resource_icon(resource)}"
+      .summary-card__title
+        h5= link_to resource.title, resource
+        h6
+          - if resource.user&.full_name&.present?
```

Comment:
@T-Dnzt Can you make this the author, not the user? It should be in `resource.metadata['creators']` I believe.